api/emails.js

```diff
@@ -0,0 +1,24 @@
+var request = require('./request');
+var urls = require('./urls');
+/**
+ * Send password reset email.
+ * @param {string} email Email address.
+ * @return {HTTPClient} The http(s) client.
+ */
+function passwordReset(email) {
+  return new Promise(function(resolve, reject) {
+    return request
+      .post({
+        url: urls.passwordReset(),
+        body: {
+          email: email
+        }
+      })
+      .then(function(res) {
+        resolve(res.body);
+      })
+      .catch(reject);
```

Comment:
This could be written as:
```js
function passwordReset(email) {
  return request
    .post({
      url: urls.passwordReset(),
      body: {
        email: email
      }
    })
    .then(function(res) {
      return res.body;
    });
}
```

And it would have the same behavior (without the extra `Promise`).